scripts/dts/gen_defines.py

```diff
@@ -435,11 +435,28 @@ def irq_name_alias(irq, cell_name):
             alias += "_" + str2ident(cell_name)
         return alias
 
+    def encode_zephyr_multi_level_irq(irq, irq_num):
+        # See doc/reference/kernel/other/interrupts.rst for details
+        # on how this encoding works
+
+        irq_ctrl = irq.controller
+        # Look for interrupt controller parent until we have none
+        while irq_ctrl.interrupts:
+            irq_num = (irq_num + 1) << 8
+            if "irq" not in irq_ctrl.interrupts[0].specifier:
+                _err("Expected binding for {!r} to have 'irq' "
```

Comment:
Should be `err()` rather than `_err()`, as it's not in edtlib.py.

Reply:
Changed.